### PR TITLE
[7.0-Backported] web tree date search

### PR DESCRIPTION
**Pull request by fossoult**

### CODE REVIEW

File: web_tree_date_search/README.rst [hunk 1]
```diff
@@ -0,0 +1,40 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License
+
+Tree Date Search
+==============================
```
Comment (elicoidal) [inline web_tree_date_search/README.rst]: Hi @fossoult  thanks for your contribution
I will check mostly format and guidelines in your PR
Can you remove extra underline and adjust with sentence length?

File: web_tree_date_search/README.rst [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+This module allow you to easily add dates range search fields on top of List View rows.
```
Comment (elicoidal) [inline web_tree_date_search/README.rst]: s/allow/allows
s/on top of List View rows./on top of the tree view.

File: web_tree_date_search/README.rst [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+These dates fields can be used in combination with the Search window.
+
+How to do ?
```
Comment (elicoidal) [inline web_tree_date_search/README.rst]: # usage

Comment (elicoidal) [inline web_tree_date_search/README.rst]: Add:
This module requires developer skills to be used.

File: web_tree_date_search/README.rst [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+Add a new 'dates_filter' key and a list of fields name in the action context:
+{'dates_filter': ['start']}
+or several dates:
+{'dates_filter': ['start','stop',..]}
+
+.. image:: web_tree_date_search/static/src/img/demo.png
+.. image:: web_tree_date_search/static/src/img/demo1.png
+
+Credits
```
Comment (elicoidal) [inline web_tree_date_search/README.rst]: I miss
- the bug tracker section
- Try me on Runbot section

File: web_tree_date_search/README.rst [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
+Author
+------
+* Thomas Fossoul, Noviat <thomas.fossoul@noviat.com>
+
+Maintainer
+----------
+.. image:: http://odoo-community.org/logo.png
+   :alt: Odoo Community Association
+   :target: http://odoo-community.org
```
Comment (elicoidal) [inline web_tree_date_search/README.rst]: https

File: web_tree_date_search/README.rst [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+
+This module is maintained by the OCA.
+
+OCA, or the Odoo Community Association, is a nonprofit organization whose
+mission is to support the collaborative development of Odoo features and
+promote its widespread use.
+
+To contribute to this module, please visit http://odoo-community.org.
```
Comment (elicoidal) [inline web_tree_date_search/README.rst]: https

File: web_tree_date_search/__init__.py [hunk 7]
```diff
@@ -0,0 +1,18 @@
+##############################################################################
```
Comment (elicoidal) [inline web_tree_date_search/__init__.py]: Please use simple headers following OCA templates
https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L3

**Comment by fossoult** [comment]
@elicoidal  Should I remove the line 33 to pass the test ?

.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
   :alt: Try me on Runbot
   :target: https://runbot.odoo-community.org/runbot/162/7.0

**Comment by elicoidal** [comment]
cc @pedrobaeza

**Comment by fossoult** [comment]
I can update with " # noqa: skip 79 char because I can't change the URL"

**Comment by fossoult** [comment]
Can you merge it plz ?

**Comment by hbrunn** [comment]
@fossoult when backporting stuff, pick the original commits as explained in https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-8.0#how with the necessary changes

File: web_tree_date_search/__openerp__.py [hunk 8]
```diff
@@ -0,0 +1,86 @@
...
+To contribute to this module, please visit http://odoo-community.org.
+    """,
+    'depends': [
+        'web',
+    ],
+    'js': [
+        'static/src/js/search.js',
+    ],
+    'qweb': [
+        'static/src/xml/*.xml',
```
Comment (rousseldenis) [inline web_tree_date_search/__openerp__.py]: @fossoult Please name your file(s). Not very safe

**Review by rousseldenis — CHANGES_REQUESTED** [review]
code review. Minor comment

**Review by rousseldenis — COMMENTED** [review]
@fossoult Have you done all modifications requested by @elicoidal and @hbrunn ?

**Review by rousseldenis — COMMENTED** [review]
@tfossoulw Can you check lint errors in javascript and solve them?

**Comment by gurneyalex** [comment]
Hello @fossoult 

I'm having issues on runbot with this PR. Would you mind making a new commit and pushing

**Comment by fossoult** [comment]
@gurneyalex I relaunch the runbot and it run without any problems. Can you tell me what you saw ?

**Review by rousseldenis — APPROVED** [review]
:+1: Seems ok. Flake8 error generated by web_warning_on_save module

**Comment by gurneyalex** [comment]
ok then I consider this part of the docker migration of runbot solved. Thanks for you patience @fossoult

**Pull request closed without merging**